open_humans/management/commands/vacuum_log_bucket.py

```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
+from urllib.parse import urlparse, parse_qs
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import boto3
+from pyparsing import ZeroOrMore, Regex
+
+from data_import.models import DataFile, AWSDataFileAccessLog, NewDataFileAccessLog
+
+
+class Command(BaseCommand):
+    """
+    A management command vaccuming up file access logs and putting them in the database.
+    """
+
+    help = "Populates database with file access logs"
+
+    def handle(self, *args, **options):
+        self.stdout.write("Retreiving file access logs")
+        s3 = boto3.resource("s3")
+        bucket = s3.Bucket(settings.LOG_BUCKET)
+        for item in bucket.objects.all():
+            log_objects = item.get()["Body"].read().decode("utf-8").split("\n")
+            for log_entry in log_objects:
+                if not log_entry:
+                    continue
+                parser = ZeroOrMore(
+                    Regex(r"\[[^]]*\]") | Regex(r'"[^"]*"') | Regex(r"[^ ]+")
+                )
+                log = list(parser.parseString(log_entry))
+
+                aws_log_entry = AWSDataFileAccessLog()
+                skip_fields = ["id", "created", "data_file", "oh_data_file_access_log"]
+                fields = [
+                    field
+                    for field in aws_log_entry._meta.get_fields()
```

Comment:
I'm concerned that this is putting a lot of trust in the ordering of get_fields remaining the same, I don't think that's a good idea.

Reply:
That's a fair point.